### PR TITLE
Sync cluster api

### PR DESCRIPTION
**Pull request by ingvagabund**
- sync with cluster-api@openshift-4.0-cluster-api-0.0.0-alpha.4
- regen machinehealthchecking clientset
- bump k8s to 1.13.1

### CODE REVIEW

**Comment by ingvagabund** [comment]
```
level=error msg="Error: Error applying plan:"
level=error
level=error msg="2 errors occurred:"
level=error msg="\t* module.vpc.aws_route.igw_route: 1 error occurred:"
level=error msg="\t* aws_route.igw_route: Error creating route: timeout while waiting for state to become 'success' (timeout: 2m0s)"
level=error
level=error
level=error msg="\t* module.vpc.aws_route.to_nat_gw[4]: 1 error occurred:"
level=error msg="\t* aws_route.to_nat_gw.4: Error creating route: timeout while waiting for state to become 'success' (timeout: 2m0s)"
```

**Comment by paulfantom** [comment]
Looks like staleness detector finally works :smile:

**Comment by enxebre** [comment]
@ingvagabund can we drop the patch commit and use https://github.com/openshift/client-go/pull/98 now?

**Comment by ingvagabund** [comment]
@enxebre done, PTAL

File: Gopkg.toml [hunk 1]
```diff
@@ -6,10 +6,9 @@ required = [
   "k8s.io/code-generator/cmd/client-gen",
   "k8s.io/code-generator/cmd/lister-gen",
   "k8s.io/code-generator/cmd/informer-gen",
-  "k8s.io/code-generator/cmd/openapi-gen",
+  # "k8s.io/code-generator/cmd/openapi-gen",
```
Comment (enxebre) [inline Gopkg.toml]: why is this commented out?

Comment (ingvagabund) [reply Gopkg.toml]: `cmd/openapi-gen` is no longer part of the latest `k8s.io/code-generator` package. Keeping it here so it can be replaced/fixed later.

Comment (frobware) [reply Gopkg.toml]: We have git, we should drop this line. When I see something like this it is not clear whether it was intentional, something that did not get cleared up before PR submission, et al.

Comment (ingvagabund) [reply Gopkg.toml]: Line removed

**Comment by enxebre** [comment]
known flaky https://jira.coreos.com/browse/CLOUD-436
/retest

**Pull request merged**